src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -1075,6 +1078,71 @@ func (c controller) logClusterOperatorsStatus() {
 	}
 }
 
+// This is temporary function, till https://bugzilla.redhat.com/show_bug.cgi?id=2097041 will not be resolved,
+//that should validate router state only in case of failure
+// It will patch router to add access logs
+// It will run http router health check to see if router is healthy on host network
+func (c controller) logRouterStatus() {
+	if !c.Status.HasError() || c.HighAvailabilityMode != models.ClusterHighAvailabilityModeNone {
+		return
+	}
+	c.log.Infof("Start checking router status")
+	var cl *models.Cluster
+	var err error
+	patched := false
+	//nolint:gosec // need insecure TLS option for testing and development
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+	client := &http.Client{Transport: tr, Timeout: dnsValidationTimeout}
+	localIp, _ := utils.GetOutboundIP()
+	if localIp == "" {
+		localIp = "localhost"
+	}
+
+	var consoleUrl string
+	// run every 30 seconds router check and log the result
+	go func() {
+		_ = utils.WaitForPredicate(WaitTimeout, SummaryLogsPeriod, func() bool {
+			if consoleUrl == "" {
+				cl, err = c.ic.GetCluster(context.Background(), false)
+				if err != nil {
+					c.log.WithError(err).Errorf("Failed to get cluster %s from assisted-service", c.ClusterID)
+					return false
+				}
+			}
+
+			consoleUrl = fmt.Sprintf("https://console-openshift-console.apps.%s.%s/health", cl.Name, cl.BaseDNSDomain)
+			r, err := client.Get(consoleUrl)
+			if err != nil {
+				c.log.WithError(err).Warning("Failed to reach console")
+			} else {
+				c.log.Infof("console url status %s", r.Status)
+			}
+
+			url := fmt.Sprintf("http://%s/_______internal_router_healthz", localIp)
```

Comment:
Is this right?
All the _____?
Why not user /healthz ?

Reply:
this is what we were asked to set. Yep it is doing what it should